app/controllers/api/public/v1/subjects_controller.rb

```diff
@@ -0,0 +1,41 @@
+module API
+  module Public
+    module V1
+      class SubjectsController < API::Public::V1::ApplicationController
+        def index
+          render json: {
+            data: [
+              {
+                "id": "1",
+                "type": "Subject",
+                "attributes": {
+                  "subject_name": "Primary",
+                  "subject_code": "00",
```

Comment:
in yml they are with out prefix of `subject_`

![image](https://user-images.githubusercontent.com/470137/89399229-14648c00-d70a-11ea-8f88-b348b02a177a.png)